packages/icon/src/generated/ArrowRight.tsx

```diff
@@ -2,11 +2,13 @@
  * This is a generated file. Do not modify it manually.
  *
  * @script ./node_modules/.bin/ts-node packages/icon/scripts/build.ts
- * @checksum 089a15a9efb4b3a77dda3129c14dd2ae
+ * @checksum 1f595d19570a615b5f15aa38aa9ed64a
```

Comment:
I assume this is just regenerated on every build?

Reply:
yes